NEWS.md

```diff
@@ -18,6 +18,25 @@ purrr and dplyr:
   dplyr (#217).
 
 
+## pluck()
+
+The plucking mechanism used for indexing into data structures with
+`map()` has been extracted into the function `pluck()`. Plucking is
+often more readable to extract an element buried in a deep data
+structure. Compare this syntax-heavy extraction which reads
+non-linearly:
+
+```
+accessor(x[[1]])$foo
+```
+
+to the equivalent pluck:
+
+```
+pluck(x, 1, accessor, "foo")
```

Comment:
yup it's like an indexing-oriented composition operator. It is also a bit more explicit regarding the intent of the composition (e.g. extraction).